src/com/sun/jna/Native.java

```diff
@@ -947,6 +955,18 @@ private static void loadNativeDispatchLibrary() {
         }
 
         String libName = System.getProperty("jna.boot.library.name", "jnidispatch");
+
+        // if in static init mode, load the library by name and force-run the initializer
+        if (!Boolean.getBoolean("jna.skipStatic")) {
+            try {
+                System.loadLibrary(libName);
+                assert isStaticEnabled();
+                assert initializeStatic() == 0x00010004;
```

Comment:
This can be dropped now, and, furthermore, it would cause issues if `[lib]jnidispatch.[so|dylib|dll]` already exists on the user's `java.library.path`.